### ISSUE TITLE
Is this a port?

### ISSUE THREAD

**Issue by xahon** [omgwtfgames/csg.cs#5]
Is that a port of https://github.com/evanw/csg.js/blob/master/csg.js ?

**Comment by pansapiens** [comment]
Yes, essentially. It's a port of https://github.com/timknip/csg.as which is a port of evanw/csg.js. Note the outstanding issues - if you plan on using it.
